docs/source/dev-and-test/setup-run-node.md

```diff
@@ -38,3 +38,41 @@ The BigchainDB [CONTRIBUTING.md file](https://github.com/bigchaindb/bigchaindb/b
 Ian Worrall of [Encrypted Labs](http://www.encryptedlabs.com/) wrote a document (PDF) explaining how to set up a BigchainDB (Server) dev machine on [Cloud9](https://c9.io/):
 
 [Download that document from GitHub](https://github.com/bigchaindb/bigchaindb/raw/master/docs/source/_static/cloud9.pdf)
+
+
+## Option C: Using a Local Dev Machine and Docker
+
+You need to have recent versions of [docker engine](https://docs.docker.com/engine/installation/#installation)
+and [docker-compose](https://docs.docker.com/compose/install/), and possibly
+[docker-machine](https://docs.docker.com/machine/install-machine/),`if you are
+running on Mac OS X or Windows, for instance.
+
+Build the images:
+
+```bash
+$ docker-compose build
```

Comment:
I used to include the dollar sign `$` when giving command-line commands in the documentation, but Dimi pointed out that it's more difficult to copy and paste the command if the dollar sign is there. Now I just leave out the dollar sign. It should be clear from context that this is a command-line command.